design/src/rfcs/rfc0016_body_callback_apis_v2.md

```diff
@@ -108,21 +116,26 @@ impl Crc32cCallback {
 
     fn trailers(&self) -> Result<Option<HeaderMap<HeaderValue>>, BoxError> {
```

Comment:
Seems like you could provide a default implementation for this function on the trait since it's the same for all four now.